concurrency/2.0/_index.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Jakarta Concurrency 2.0 (milestone)"
-date: 2020-05-30
+title: "Jakarta Concurrency 2.0"
+date: 2020-09-12
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Concurrency provides a specification for using concurrency from application components without compromising container integrity while still preserving the Jakarta EE platform's fundamental benefits.
 
 * [Jakarta Concurrency 2.0 Release Record](https://projects.eclipse.org/projects/ee4j.cu/releases/2.0.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Concurrency 2.0 Specification Document](./concurrency-spec-2.0-RC2.pdf) (PDF)
-* [Jakarta Concurrency 2.0 Specification Document](./concurrency-spec-2.0-RC2.html) (HTML)
+* [Jakarta Concurrency 2.0 Specification Document](./concurrency-spec-2.0.pdf) (PDF)
+* [Jakarta Concurrency 2.0 Specification Document](./concurrency-spec-2.0.html) (HTML)
 * [Jakarta Concurrency 2.0 Javadoc](./apidocs)
-* [Jakarta Concurrency 2.0 TCK]()([sig](),[sha](),[pub]())
+* [Jakarta Concurrency 2.0 TCK](https://download.eclipse.org/jakartaee/concurrency/2.0/jakarta-concurrency-tck-2.0.0.zip)([sig](https://download.eclipse.org/jakartaee/concurrency/2.0/jakarta-concurrency-tck-2.0.0.zip.sig),[sha](https://download.eclipse.org/jakartaee/concurrency/2.0/jakarta-concurrency-tck-2.0.0.zip.sha256),[pub](https://raw.githubusercontent.com/jakartaee/specification-committee/master/jakartaee-spec-committee.pub))
 * Maven coordinates
-  * [jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api:2.0.0-RC2](https://jakarta.oss.sonatype.org/service/local/repo_groups/staging/content/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-api/2.0.0-RC2/jakarta.enterprise.concurrent-api-2.0.0-RC2.jar)
+  * [jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api:2.0.0](https://jakarta.oss.sonatype.org/service/local/repo_groups/staging/content/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-api/2.0.0/jakarta.enterprise.concurrent-api-2.0.0.jar)
 
```

Comment:
This looks like staging repo. It should be "https://search.maven.org/artifact/...".

Reply:
Good catch @kazumura!
@smillidge Should be updated before we merge.